convert2rhel/subscription.py

```diff
@@ -96,14 +114,11 @@ def unregister_system():
     if ret_code != 0:
         loggerinst.info("The subscription-manager package is not installed.")
         return
+
     unregistration_cmd = ["subscription-manager", "unregister"]
     output, ret_code = utils.run_subprocess(unregistration_cmd, print_output=False)
     if ret_code != 0:
-        loggerinst.warning(
-            "System unregistration failed with return code %d and message:\n%s",
-            ret_code,
-            output,
-        )
+        raise UnregisterError("System unregistration failed with return code %d and message:\n%s" % (ret_code, output))
```

Comment:
```suggestion
        raise UnregisterError("System unregistration result:\n%s" % output)
```
Because most customers will see this warning (before registering), I'd ease out the wording.